cheatsheets/Authentication_Cheat_Sheet.md

```diff
@@ -23,49 +23,24 @@ For information on validating email addresses, please visit the [input validatio
 
 A key concern when using passwords for authentication is password strength. A "strong" password policy makes it difficult or even improbable for one to guess the password through either manual or automated means. The following characteristics define a strong password:
 
-### Password Length
+- Password Length
+    - **Minimum** length of the passwords should be **enforced** by the application. Passwords **shorter than 8 characters** are considered to be weak ([NIST SP800-63B](https://pages.nist.gov/800-63-3/sp800-63b.html)). 
+    - **Maximum** password length should not be set **too low**, as it will prevent users from creating passphrases. Typical maximum length is 128 characters.
 
-Longer passwords provide a greater combination of characters and consequently make it more difficult for an attacker to guess.
+- Do do not truncate passwords. Make sure that every character the user types in is actually included in the password. 
 
-- **Minimum** length of the passwords should be **enforced** by the application.
-    - Passwords **shorter than 8 characters** are considered to be weak ([NIST SP800-63B](https://pages.nist.gov/800-63-3/sp800-63b.html)).
+- Allow usage of **all** characters including unicode and whitespaces. There should be no password composition rules limiting the type of characters permitted.
 
-While minimum length enforcement may cause problems with memorizing passwords among some users, applications should encourage them to set *passphrases* (sentences or combination of words) that can be much longer than typical passwords and yet much easier to remember.
+- Make sure that there are no periodic credential rotation or password history requirements.
```

Comment:
Even a rotation once a year?

Reply:
Why are you giving this recommendation? What if it is a critical service? It's true that rotation is being dropped, but only for the usage of 2FA, other means of authentication, etc.

Reply:
This point is based on [ASVS](https://github.com/OWASP/ASVS/blob/master/4.0/en/0x11-V2-Authentication.md#v21-password-security-requirements). 
[NIST](https://pages.nist.gov/800-63-3/sp800-63b.html), [Troy](https://www.troyhunt.com/passwords-evolved-authentication-guidance-for-the-modern-era/) and [Microsoft](https://blogs.technet.microsoft.com/secguide/2019/04/24/security-baseline-draft-for-windows-10-v1903-and-windows-server-v1903/) also agree on that.
We can change it or add some more details to it - for example that this apples only for memorized passwords by humans etc.

Reply:
I'd replace this sentence then with something similar to this one:
`Ensure credential rotation when a password leak, or at the time of compromise identification`
Something to this sounds better than saying `Make sure that ...`